README.md

```diff
@@ -93,14 +93,14 @@ Themis is available for the following languages/platforms, refer to [language ho
 
 # Availability
 
-Themis supports following CPU architectures: x86_64/i386, ARM, various Android architectures.
+Themis supports following CPU architectures: x86_64/i386, ARM, Apple Silicon (ARM64), various Android architectures.
 
 We build and verify Themis on the latest stable OS versions:
 
   - Debian (9, 10), CentOS (7, 8), Ubuntu (16.04, 18.04, 20.04)
   - macOS (10.12–10.15, 11)
   - Android (4–11)
-  - iOS and iPadOS (9–14beta),
+  - iOS (10–14) <img src="https://img.shields.io/github/issues-pr-closed/cossacklabs/themis/Bitcode%20%F0%9F%90%99?color=%23cceaff&label=Bitcode-related%20issues" alt="Bitrise"></img>
```

Comment:
Badges are typically links, so I'd suggest
```suggestion
  - iOS (10–14) [![Closed pull requests with Bitcode-related changes](https://img.shields.io/github/issues-pr-closed/cossacklabs/themis/Bitcode%20%F0%9F%90%99?color=informational&label=Bitcode-related%20issues)](https://github.com/cossacklabs/themis/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Bitcode+%F0%9F%90%99%22)
```

Rendered:

  - Android (4–11)
  - iOS (10–14) [![Closed pull requests with Bitcode-related changes](https://img.shields.io/github/issues-pr-closed/cossacklabs/themis/Bitcode%20%F0%9F%90%99?color=informational&label=Bitcode-related%20issues)](https://github.com/cossacklabs/themis/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Bitcode+%F0%9F%90%99%22)
  - Windows (experimenal MSYS2 support)

I don't quite like how the badge is not aligned with the text baseline... Well, at least in Firefox.

Maybe this looks better? (With `<sub>` around the badge.)

  - Android (4–11)
  - iOS (10–14) <sub>[![Closed pull requests with Bitcode-related changes](https://img.shields.io/github/issues-pr-closed/cossacklabs/themis/Bitcode%20%F0%9F%90%99?color=informational&label=Bitcode-related%20issues)](https://github.com/cossacklabs/themis/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Bitcode+%F0%9F%90%99%22)</sub>
  - Windows (experimenal MSYS2 support)

But it's a hack and it still breaks interline spacing :(

Reply:
Granted, it's a nice joke for a chat 😏 But I don't feel this looks nice enough here, or that our users share or empathize with the maintainers' pain of supporting Bitcode. If you'd still like to keep it, let's look for a better place for those vanity metrics? How about the **Languages** section, right below the Greater Table of Platforms?

Reply:
Listening to your advice, I've moved this badge to the very bottom of a page, giving it more like an easter egg flavour.